diff --git a/src/lib.rs b/src/lib.rs
index 4d2e4ea..2a097ec 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -255,6 +255,7 @@ cfg_if! {
         // include the Nintendo Switch (which is target_arch = "aarch64").
         all(target_os = "horizon", target_arch = "arm"),
         target_os = "cygwin",
+        target_os = "slopos",
     ))] {
         mod util_libc;
         #[path = "getrandom.rs"] mod imp;
diff --git a/src/util_libc.rs b/src/util_libc.rs
index f488b41..57af6b2 100644
--- a/src/util_libc.rs
+++ b/src/util_libc.rs
@@ -11,7 +11,7 @@ use libc::c_void;
 cfg_if! {
     if #[cfg(any(target_os = "netbsd", target_os = "openbsd", target_os = "android", target_os = "cygwin"))] {
         use libc::__errno as errno_location;
-    } else if #[cfg(any(target_os = "linux", target_os = "emscripten", target_os = "hurd", target_os = "redox", target_os = "dragonfly"))] {
+    } else if #[cfg(any(target_os = "linux", target_os = "emscripten", target_os = "hurd", target_os = "redox", target_os = "dragonfly", target_os = "slopos"))] {
         use libc::__errno_location as errno_location;
     } else if #[cfg(any(target_os = "solaris", target_os = "illumos"))] {
         use libc::___errno as errno_location;
