diff --git a/src/socket.rs b/src/socket.rs
index defbb1e..d54c288 100644
--- a/src/socket.rs
+++ b/src/socket.rs
@@ -972,7 +972,7 @@ impl Socket {
     /// For more information about this option, see [`set_out_of_band_inline`].
     ///
     /// [`set_out_of_band_inline`]: Socket::set_out_of_band_inline
-    #[cfg(not(any(target_os = "redox", target_os = "wasi")))]
+    #[cfg(not(any(target_os = "redox", target_os = "slopos", target_os = "wasi")))]
     pub fn out_of_band_inline(&self) -> io::Result<bool> {
         unsafe {
             getsockopt::<c_int>(self.as_raw(), sys::SOL_SOCKET, sys::SO_OOBINLINE)
@@ -986,7 +986,7 @@ impl Socket {
     /// receive data stream. Otherwise, out-of-band data is passed only when the
     /// `MSG_OOB` flag is set during receiving. As per RFC6093, TCP sockets
     /// using the Urgent mechanism are encouraged to set this flag.
-    #[cfg(not(any(target_os = "redox", target_os = "wasi")))]
+    #[cfg(not(any(target_os = "redox", target_os = "slopos", target_os = "wasi")))]
     pub fn set_out_of_band_inline(&self, oob_inline: bool) -> io::Result<()> {
         unsafe {
             setsockopt(
@@ -1422,6 +1422,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "fuchsia",
         target_os = "nto",
         target_os = "espidf",
@@ -1462,6 +1463,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "fuchsia",
         target_os = "nto",
         target_os = "espidf",
@@ -1684,6 +1686,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "solaris",
         target_os = "haiku",
         target_os = "nto",
@@ -1718,6 +1721,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "solaris",
         target_os = "haiku",
         target_os = "nto",
@@ -2096,6 +2100,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "solaris",
         target_os = "haiku",
         target_os = "hurd",
@@ -2123,6 +2128,7 @@ impl Socket {
         target_os = "netbsd",
         target_os = "openbsd",
         target_os = "redox",
+        target_os = "slopos",
         target_os = "solaris",
         target_os = "haiku",
         target_os = "hurd",
diff --git a/src/sys/unix.rs b/src/sys/unix.rs
index be9b6da..14a4340 100644
--- a/src/sys/unix.rs
+++ b/src/sys/unix.rs
@@ -132,7 +132,7 @@ pub(crate) use libc::{
     target_os = "horizon"
 )))]
 pub(crate) use libc::MSG_TRUNC;
-#[cfg(not(any(target_os = "redox", target_os = "wasi")))]
+#[cfg(not(any(target_os = "redox", target_os = "slopos", target_os = "wasi")))]
 pub(crate) use libc::SO_OOBINLINE;
 // Used in `Socket`.
 #[cfg(not(target_os = "nto"))]
@@ -167,6 +167,7 @@ pub(crate) use libc::IPV6_RECVHOPLIMIT;
     target_os = "netbsd",
     target_os = "openbsd",
     target_os = "redox",
+    target_os = "slopos",
     target_os = "solaris",
     target_os = "haiku",
     target_os = "espidf",
@@ -193,6 +194,7 @@ pub(crate) use libc::IP_HDRINCL;
     target_os = "netbsd",
     target_os = "openbsd",
     target_os = "redox",
+    target_os = "slopos",
     target_os = "solaris",
     target_os = "haiku",
     target_os = "hurd",
@@ -249,6 +251,7 @@ pub(crate) use libc::{
     target_os = "netbsd",
     target_os = "openbsd",
     target_os = "redox",
+    target_os = "slopos",
     target_os = "fuchsia",
     target_os = "nto",
     target_os = "espidf",
@@ -397,6 +400,7 @@ const TCP_CA_NAME_MAX: usize = 16;
         )
     ),
     target_os = "android",
+    target_os = "slopos",
 ))]
 type IovLen = usize;
 
@@ -536,6 +540,7 @@ impl_debug!(
     libc::SOCK_RAW,
     #[cfg(not(any(
         target_os = "redox",
+        target_os = "slopos",
         target_os = "haiku",
         target_os = "espidf",
         target_os = "wasi",
